### PR TITLE
Upgrade @guardian/commercial@14.4.1

### PR DESCRIPTION
**Pull request by arelra**
## What does this change?

Upgrade @guardian/commercial@14.4.0

Release page here: https://github.com/guardian/commercial/releases

This includes an update that removes the `guardian/prebid.js` dependency as a GitHub reference in favour of a published package @guardian/prebid.js.

The previous GitHub reference caused issues for Yarn, Snyk and Dependabot

Wait for approval from @sndrs before merging

### CODE REVIEW

File: pnpm-lock.yaml [hunk 1]
```diff
@@ -4786,6 +4786,33 @@ packages:
     engines: {node: '>=16'}
     dev: false
 
+  /@guardian/prebid.js@8.34.0(tslib@2.6.2)(typescript@5.3.3):
```
Comment (sndrs) [inline pnpm-lock.yaml:4789]: love it, thanks @arelra (let's hope it works!)

**Review by sndrs — APPROVED** [review]
100% approve, thanks for looking into it

**Comment by sndrs** [comment]
> actualyl could you give me 5 mins before merging?

ok all good

**Review by mxdvl — APPROVED** [review]
👏 – glad you sorted that gnarly uppercase `P` 😉

**Pull request merged**